icons/gitkraken/gitkraken-original-wordmark.svg

```diff
@@ -0,0 +1 @@
 d="M13.635 109.144c-5.325 0-9.611 3.897-9.611 9.352a9.352 9.352 0 0 0 9.548 9.418c3.247 0 7.144-1.623 8.832-5.39a11.3 11.3 0 0 0 .78-4.612H12.207v2.988h6.819a5.196 5.196 0 0 1-1.883 2.729 6.235 6.235 0 0 1-3.637 1.103c-2.857 0-5.846-2.144-5.846-6.236a5.91 5.91 0 0 1 5.846-6.17 5.26 5.26 0 0 1 3.506 1.234 6.04 6.04 0 0 1 1.365 1.494h4.092c-1.17-3.312-4.742-5.91-8.834-5.91zm12.21.326v2.856h3.182v-2.856zm7.468 0v4.676H31.04v2.467h2.272v10.977h3.183v-10.977h2.338v-2.467h-2.338v-4.676zm8.119 0v18.12h3.7v-8.12l6.042 8.12h4.416l-7.209-9.094 6.56-9.026H51.11l-5.977 8.377v-8.377zm41.435 0v18.12h3.182v-6.69l4.871 6.69h3.637l-5.586-7.534 5.586-5.91h-3.832l-4.676 5.13v-9.806zm-18.77 4.286a3.832 3.832 0 0 0-3.702 2.142v-1.752h-3.051v13.444h3.182v-7.274c0-2.273 1.168-3.379 3.44-3.379h.132zm7.534 0c-3.767 0-6.82 3.248-6.82 7.08s2.533 7.142 6.82 7.142a5.066 5.066 0 0 0 4.547-2.338v1.95h3.182v-13.444h-3.182v1.947a5.066 5.066 0 0 0-4.547-2.337zm30.07 0a6.82 6.82 0 0 0-4.937 2.209 7.079 7.079 0 0 0 .39 10.066 7.014 7.014 0 0 0 10.002-.26 6.69 6.69 0 0 0 1.819-3.637 7.728 7.728 0 0 0-.586-4.287l-2.272 2.405a3.507 3.507 0 0 1-.91 3.57 3.767 3.767 0 0 1-5.13.584l7.468-7.922a7.404 7.404 0 0 0-5.844-2.728zm17.08 0c-2.403-.065-3.57 1.105-4.156 2.209v-1.819h-2.988v13.444h3.183v-7.405c0-1.884.778-3.441 3.051-3.441s2.924 1.558 2.924 3.572v7.274h3.182v-8.12c0-2.338-.78-5.714-5.196-5.714zm-92.935.39v13.444h3.181v-13.444zm75.576 2.389a3.637 3.637 0 0 1 2.162.598l-5.26 5.521a3.572 3.572 0 0 1 .455-4.807 3.637 3.637 0 0 1 2.643-1.312zm-29.336.209a3.897 3.897 0 0 1 3.701 2.338 3.767 3.767 0 0 1 .39 1.883 4.806 4.806 0 0 1-.519 1.882 3.702 3.702 0 0 1-3.572 2.145 4.027 4.027 0 0 1-4.092-4.092 4.027 4.027 0 0 1 4.092-4.156z" class="cls-1"/><path fill="#179287" d="M119.3 23.288a2.338 2.338 0 0 0-4.545.844 1.624 1.624 0 0 0 .13.78 53.644 53.644 0 0 1 3.442 19.159 54.424 54.424 0 0 1-47.215 53.904V72.58a39.097 39.097 0 0 0 4.74-1.234V92.65A50.007 50.007 0 0 0 88.195.298a2.403 2.403 0 0 0-3.313 1.039 2.923 2.923 0 0 0-.194 1.039 2.338 2.338 0 0 0 1.169 2.078 45.266 45.266 0 0 1-5.262 81.766v-18.9a7.274 7.274 0 0 0 5.066-6.883 7.144 7.144 0 0 0-3.572-6.3c1.688-16.301 9.157-12.015 9.157-17.21v-2.988C91.245 26.08 73.19.752 64.748.103h-1.56C54.746.753 36.69 26.08 36.69 33.939v2.988c0 5.195 7.47.909 9.158 17.21a7.274 7.274 0 0 0 1.558 13.184V86.22a45.266 45.266 0 0 1-5.325-81.766 2.338 2.338 0 0 0 .974-3.117 2.273 2.273 0 0 0-2.143-1.3 2.143 2.143 0 0 0-1.104.26 50.007 50.007 0 0 0 12.34 92.352V71.347a39.097 39.097 0 0 0 4.74 1.234v25.394A54.424 54.424 0 0 1 9.609 44.07a55.073 55.073 0 0 1 3.443-19.16 2.338 2.338 0 0 0-1.234-2.987 2.598 2.598 0 0 0-.975-.194 2.403 2.403 0 0 0-2.208 1.558 59.1 59.1 0 0 0 52.93 79.817V73.23h4.806v29.874a59.035 59.035 0 0 0 56.697-59.034 57.866 57.866 0 0 0-3.767-20.783zM74.88 56.995a4.87 4.87 0 1 1 6.884 6.884 4.87 4.87 0 1 1-6.884-6.884zm-21.822 6.884a4.806 4.806 0 0 1-6.82 0 4.87 4.87 0 0 1 0-6.884 4.806 4.806 0 0 1 6.82 0 4.87 4.87 0 0 1 0 6.884z"/></svg>
```

Comment:
Found just two small things
- Add newline at EOF
- remove  `class="cls-1"` since it has no effect here (?)

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M13.635 109.144c-5.325 0-9.611 3.897-9.611 9.352a9.352 9.352 0 0 0 9.548 9.418c3.247 0 7.144-1.623 8.832-5.39a11.3 11.3 0 0 0 .78-4.612H12.207v2.988h6.819a5.196 5.196 0 0 1-1.883 2.729 6.235 6.235 0 0 1-3.637 1.103c-2.857 0-5.846-2.144-5.846-6.236a5.91 5.91 0 0 1 5.846-6.17 5.26 5.26 0 0 1 3.506 1.234 6.04 6.04 0 0 1 1.365 1.494h4.092c-1.17-3.312-4.742-5.91-8.834-5.91zm12.21.326v2.856h3.182v-2.856zm7.468 0v4.676H31.04v2.467h2.272v10.977h3.183v-10.977h2.338v-2.467h-2.338v-4.676zm8.119 0v18.12h3.7v-8.12l6.042 8.12h4.416l-7.209-9.094 6.56-9.026H51.11l-5.977 8.377v-8.377zm41.435 0v18.12h3.182v-6.69l4.871 6.69h3.637l-5.586-7.534 5.586-5.91h-3.832l-4.676 5.13v-9.806zm-18.77 4.286a3.832 3.832 0 0 0-3.702 2.142v-1.752h-3.051v13.444h3.182v-7.274c0-2.273 1.168-3.379 3.44-3.379h.132zm7.534 0c-3.767 0-6.82 3.248-6.82 7.08s2.533 7.142 6.82 7.142a5.066 5.066 0 0 0 4.547-2.338v1.95h3.182v-13.444h-3.182v1.947a5.066 5.066 0 0 0-4.547-2.337zm30.07 0a6.82 6.82 0 0 0-4.937 2.209 7.079 7.079 0 0 0 .39 10.066 7.014 7.014 0 0 0 10.002-.26 6.69 6.69 0 0 0 1.819-3.637 7.728 7.728 0 0 0-.586-4.287l-2.272 2.405a3.507 3.507 0 0 1-.91 3.57 3.767 3.767 0 0 1-5.13.584l7.468-7.922a7.404 7.404 0 0 0-5.844-2.728zm17.08 0c-2.403-.065-3.57 1.105-4.156 2.209v-1.819h-2.988v13.444h3.183v-7.405c0-1.884.778-3.441 3.051-3.441s2.924 1.558 2.924 3.572v7.274h3.182v-8.12c0-2.338-.78-5.714-5.196-5.714zm-92.935.39v13.444h3.181v-13.444zm75.576 2.389a3.637 3.637 0 0 1 2.162.598l-5.26 5.521a3.572 3.572 0 0 1 .455-4.807 3.637 3.637 0 0 1 2.643-1.312zm-29.336.209a3.897 3.897 0 0 1 3.701 2.338 3.767 3.767 0 0 1 .39 1.883 4.806 4.806 0 0 1-.519 1.882 3.702 3.702 0 0 1-3.572 2.145 4.027 4.027 0 0 1-4.092-4.092 4.027 4.027 0 0 1 4.092-4.156z"/><path fill="#179287" d="M119.3 23.288a2.338 2.338 0 0 0-4.545.844 1.624 1.624 0 0 0 .13.78 53.644 53.644 0 0 1 3.442 19.159 54.424 54.424 0 0 1-47.215 53.904V72.58a39.097 39.097 0 0 0 4.74-1.234V92.65A50.007 50.007 0 0 0 88.195.298a2.403 2.403 0 0 0-3.313 1.039 2.923 2.923 0 0 0-.194 1.039 2.338 2.338 0 0 0 1.169 2.078 45.266 45.266 0 0 1-5.262 81.766v-18.9a7.274 7.274 0 0 0 5.066-6.883 7.144 7.144 0 0 0-3.572-6.3c1.688-16.301 9.157-12.015 9.157-17.21v-2.988C91.245 26.08 73.19.752 64.748.103h-1.56C54.746.753 36.69 26.08 36.69 33.939v2.988c0 5.195 7.47.909 9.158 17.21a7.274 7.274 0 0 0 1.558 13.184V86.22a45.266 45.266 0 0 1-5.325-81.766 2.338 2.338 0 0 0 .974-3.117 2.273 2.273 0 0 0-2.143-1.3 2.143 2.143 0 0 0-1.104.26 50.007 50.007 0 0 0 12.34 92.352V71.347a39.097 39.097 0 0 0 4.74 1.234v25.394A54.424 54.424 0 0 1 9.609 44.07a55.073 55.073 0 0 1 3.443-19.16 2.338 2.338 0 0 0-1.234-2.987 2.598 2.598 0 0 0-.975-.194 2.403 2.403 0 0 0-2.208 1.558 59.1 59.1 0 0 0 52.93 79.817V73.23h4.806v29.874a59.035 59.035 0 0 0 56.697-59.034 57.866 57.866 0 0 0-3.767-20.783zM74.88 56.995a4.87 4.87 0 1 1 6.884 6.884 4.87 4.87 0 1 1-6.884-6.884zm-21.822 6.884a4.806 4.806 0 0 1-6.82 0 4.87 4.87 0 0 1 0-6.884 4.806 4.806 0 0 1 6.82 0 4.87 4.87 0 0 1 0 6.884z"/></svg>

```